src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -23,6 +23,38 @@ using namespace cldnn;
 #define LOG_NODE_REMOVAL(id)      GPU_DEBUG_LOG_PASS << "Remove node: " << (id) << std::endl;
 #define LOG_NODE_REPLACEMENT(id)  GPU_DEBUG_LOG_PASS << "Replace node: " << (id) << std::endl;
 
+static inline bool check_redundant_1d_reorder(const reorder_node& r_node) {
+    auto o_layout = r_node.get_output_layout();
+    auto i_layout = r_node.input().get_output_layout();
+
+    if (r_node.has_fused_primitives() == false && !i_layout.data_padding && !o_layout.data_padding &&
+        i_layout.data_type == o_layout.data_type && i_layout.count() == o_layout.count()) {
+        // Get inner block size
+        auto input_block_size = format::traits(i_layout.format).block_sizes;
+        auto block_size = format::traits(o_layout.format).block_sizes;
+
+        // Check  1d reorder
+        if (static_cast<size_t>(i_layout.feature()) == o_layout.count() && i_layout.feature() == o_layout.feature() &&
+            block_size.size() == 1) {
+            return true;
+        }
+
+        // Acceptable if a feature size of input 'byxf' fits to output's inner block size of 'b_fs_yx_fsv?'
+        if (i_layout.format == format::byxf && (o_layout.format == format::b_fs_yx_fsv16 ||  o_layout.format == format::b_fs_yx_fsv32) &&
+            block_size.size() == 1 && i_layout.feature() == block_size.at(0).second &&
+            i_layout.batch() == o_layout.batch()) {
+            // each spatial axis should be same
+            for (size_t i = 0 ; i < i_layout.get_spatial_rank() ; i++) {
+                if (i_layout.spatial(i) != o_layout.spatial(i))
+                    return false;
+            }
+
+            return true;
+        }
+    }
+
+    return false;
+}
```

Comment:
could you explain why you decide to introduce a new function instead of existing function(`are_layouts_identical`)?

Reply:
It moved to are_layouts_identical